DependencyInjection/DoctrineExtension.php

```diff
@@ -863,6 +863,11 @@ public function getConfiguration(array $config, ContainerBuilder $container) : C
         return new Configuration($container->getParameter('kernel.debug'));
     }
 
+    protected function getMetadataDriverClass(string $driverType): string
+    {
+        return '%'.$this->getObjectManagerElementName('metadata.'.$driverType.'.class%');
```

Comment:
I have no idea what this is about, but having one percent sign inside the parenthesis, and the other percent sign outside does not feel right.

Reply:
I just looked at the methdod definition, don't mind me, it should work.